spring-core/src/main/java/org/springframework/util/MimeType.java

```diff
@@ -268,7 +268,8 @@ public boolean isWildcardType() {
 	 * @return whether the subtype is a wildcard
 	 */
 	public boolean isWildcardSubtype() {
-		return WILDCARD_TYPE.equals(getSubtype()) || getSubtype().startsWith("*+");
+		String subtype = getSubtype();
+		return WILDCARD_TYPE.equals(subtype) || subtype.startsWith("*+");
```

Comment:
Aside from extracting a variable, how does this improve performance?

Reply:
see response in MimeTypeUtils

Reply:
Yes I see it there, and thanks for clarifying, but the change seems unrelated. Whether the subtype is accessed through a local variable or not, it is the same String. Am I missing anything?